frontend/XCITE/SendCoins/SendDiode.qml

```diff
@@ -276,17 +277,18 @@ Controls.Diode {
             Layout.fillWidth: true
             Layout.leftMargin: 22
             Layout.rightMargin: 22
-            height: 1
+            height: (xcite.height > 850) ? 1 : 0
             color: "#535353"
         }
 
         // Send Payment
         Controls.ButtonModal {
             Layout.fillWidth: true
-            Layout.topMargin: 35
+            Layout.topMargin: xcite.height > 850 ? 35 : -75
+            width: Screen.height < 850 ? 500 : -200
```

Comment:
This is great :) being really picky now, but let's make this as nice as possible! 
There's a 1px 'gap' at 850 in your conditionals: `> 850` and `< 850`.

This manifests in a couple of ways:
- You can wind up with the full sized button in the top area:

<img width="838" alt="screen shot 2018-02-24 at 12 53 08" src="https://user-images.githubusercontent.com/36036056/36630638-c306807e-1961-11e8-8959-f33258286375.png">

- Jumps around a little when slowly shrinking the window:

https://gyazo.com/9f4be63bcc847413794d1fc1a77e7136

Mind taking a look to see if you can just make it 'snap' between bit and small mode?

Thanks!

Reply:
I like this solution, seems to work on my end. Let me know.